src/main/java/hudson/plugins/spotinst/cloud/SpotinstInstanceWeight.java

```diff
@@ -71,13 +128,41 @@ public ListBoxModel doFillAwsInstanceTypeFromAPIItems() {
             return retVal;
         }
 
-        public FormValidation doCheckAwsInstanceTypeFromAPI() {
+        public AutoCompletionCandidates doAutoCompleteAwsInstanceTypeFromAPISearch(@QueryParameter String value) {
+            AutoCompletionCandidates c = new AutoCompletionCandidates();
```

Comment:
coding conventions, change to: AutoCompletionCandidates retVal = new AutoCompletionCandidates();